tests/php/Controllers/Root/RootController/RootControllerDevActionTest.php

```diff
@@ -443,4 +443,21 @@ public function testDevActionIsTabbarStatusEqualHomeWhenIsDevParam()
             'error:アサイン情報としてtabbarStatusはdevではない'
         );
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)